assets/pages/oneoff-contributions/components/oneoffInlineContributionsPayment.jsx

```diff
@@ -85,22 +85,22 @@ function mapDispatchToProps(dispatch: Dispatch<OneOffCheckoutAction | StripeInli
 
 // If the form is valid, calls the given callback, otherwise sets an error.
 function formValidation(
-  isFormEmpty: boolean,
+  areAnyRequiredFieldsEmpty: boolean,
```

Comment:
Out of scope for this PR, but I feel a clearer and more general abstraction for form validation could be to pass in the required fields and have them checked for truthiness inside the validation function.

Alternatively, an ideal solution would be to use the HTML5 constraint validation API!

https://developer.mozilla.org/en-US/docs/Learn/HTML/Forms/Form_validation#Validating_forms_using_JavaScript

https://caniuse.com/#search=constraint%20validation%20API